x-pack/legacy/plugins/maps/public/components/layer_addpanel/source_select/_source_select.scss

```diff
@@ -2,6 +2,9 @@
   // EUITODO: Fix horizontal layout so it works with any size icon
   .euiCard__content {
     padding-top: 0 !important;
+    .euiBetaBadge {
```

Comment:
@aaronjcaldwell Please don't override EUI classes directly. The original selector is because there's a layout bug in EUI and this is a temp fix. Your CSS is an override/layout enhancement and you should add a new custom class to the EuiBetaBadge and use SASS variables.

Reply:
It also should not use `!important`